python-packages/securesync/templates/securesync/register_public_key_client.html

```diff
@@ -99,53 +74,46 @@
 
 
     {% if unregistered %}
-    <div class="chunk-container">
-        <h2>{% trans "What is device registration?" %}</h2>
-        <p class="normal-size" style="padding: 0px 40px 0px 40px;">
+    <div class="row">
+        <h2 class="col-md-12">{% trans "What is device registration?" %}</h2>
+        <p class="col-md-10" style="margin-left:40px">
```

Comment:
Note: as a general rule avoid in-line CSS and Javascript, this includes
inside script tags on the page. If in doubt, put it in khan-lite.css with a
comment block above and below, stating what it is for. As these styles are
applied across the site, you should also make sure your class names and
selectors are specific.

On Wed, 8 Apr 2015 11:34 jtamiace <notifications@github.com> wrote:

> In
> python-packages/securesync/templates/securesync/register_public_key_client.html
> <https://github.com/learningequality/ka-lite/pull/3465#discussion_r27999302>
> :
>
> > @@ -99,53 +74,46 @@
> >
> >
> >      {% if unregistered %}
> > -    <div class="chunk-container">
> > -        <h2>{% trans "What is device registration?" %}</h2>
> > -        <p class="normal-size" style="padding: 0px 40px 0px 40px;">
> > +    <div class="row">
> > +        <h2 class="col-md-12">{% trans "What is device registration?" %}</h2>
> > +        <p class="col-md-10" style="margin-left:40px">
>
> There are several instances of this on the page. Other than that, great
> job rewriting the table!! :)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/learningequality/ka-lite/pull/3465/files#r27999302>.
>